dioptas/controller/MaskController.py

```diff
@@ -370,35 +374,56 @@ def cosmic_btn_click(self):
 
     def save_mask_btn_click(self):
         img_filename, _ = os.path.splitext(os.path.basename(self.model.img_model.filename))
-        filename = save_file_dialog(self.widget, "Save mask data",
-                                    os.path.join(self.model.working_directories['mask'],
-                                                 img_filename + '.mask'),
-                                    filter='Mask (*.mask)')
+        filename, selected_filter = QtWidgets.QFileDialog.getSaveFileName(
+            self.widget,
+            caption="Save mask data",
+            directory=os.path.join(self.model.working_directories['mask'],
+                                   img_filename + '.mask'),
+            filter=';;'.join([
+                self.DEFAULT_MASK_FILTER,
```

Comment:
For completeness, shouldn't we add?

```suggestion
            filter=';;'.join([
                self.DEFAULT_MASK_FILTER,
                "Mask (*.npy)",
                "Mask (*.edf)",
```